lib/node/runner/api/resource/transaction.go

```diff
@@ -27,6 +31,7 @@ func (t Transaction) GetMap() hal.Entry {
 		"sequence_id":     t.bt.SequenceID,
 		"created":         t.bt.Created,
 		"operation_count": len(t.bt.Operations),
+		"operations":      t.tx.B.Operations,
```

Comment:
the shape of `operation` in the transaction resource may differ from `operation resource`.
But I think that the `operation resource` will be deprecated and relevant APIs no longer under maintenance.